yamlReader.py

```diff
@@ -0,0 +1,167 @@
+import yaml
+import yamlordereddictloader
+import start
+
+yamlName = start.yamlName()
+
+
+def getDefault():
+    defaultDay = -1
```

Comment:
It is weird that a variable named `defaultDay` ever takes a value of -1.

`null` should be more appropriate